setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
```

Comment:
I image that it's not possible for this file to live under the `py/` directory?

Reply:
Yes, we actually can place this file to the `py/` directory, will do.